### DISCUSSION TITLE
geyser support

### DISCUSSION THREAD

**Discussion by ghost** [xenondevs/Nova#336] [Q&A] [answered]
hi
how to use nova through geyser addon for bedrock and java?
where can I get mspac?

**Comment by ByteZ1337** [comment, marked as answer]
Nova is not compatible with any Bedrock bridges.

**Reply by ghost** [reply]
Hi @ByteZ1337 and if I convert and install mpack, it won’t work?
Is there any plan to support bedrock?

**Reply by ByteZ1337** [reply]
Nova is using bugs in the Java edition to achieve a lot of the features in the resource pack, so no. We're also not planning to add support since bedrock already has a native way to add custom stuff.

**Reply by ghost** [reply]
Thank you for explanation
